ui/app/components/tool-hash.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class HashTool extends Component {
+  @action
+  onClear() {
+    this.args.onClear();
```

Comment:
for all new components the `onClear()` action is passed through to the child as a property and then called again via an action.